### PR TITLE
Add wp_get_attachment_alt_text function to get image alt text

### PR DESCRIPTION
**Pull request by peyman-mohamadpour**
Add a twig function to get image alt text from `_wp_attachment_image_alt` post meta key.

### CODE REVIEW

File: Wordpress/Helper/AttachmentHelper.php [hunk 1]
```diff
@@ -78,4 +78,30 @@ public function getAttachmentUrl(Post $post, $size = 'post-thumbnail')
...
+        // Switch to correct blog
+        $originBlogId = $this->getManager()->getBlogId();
+        if ($originBlogId !== $post->getBlogId()) {
+            $this->managerRegistry->setCurrentBlogId($post->getBlogId());
+        }
+
+        $metadata = $this->getManager()->getRepository('KayueWordpressBundle:PostMeta')->findOneBy([
+            'post' => $post,
+            'key' => '_wp_attachment_image_alt',
+        ]);
```
Comment (kayue) [inline Wordpress/Helper/AttachmentHelper.php]: If I remember correctly `findOneBy` might throw `NoResultException` exception if no record found. 
A try catch block here would be nice.

**Comment by peyman-mohamadpour** [comment]
Tried to log the exception and return an empty string in case of possibel exception thrown during `fundOneBy`

**Comment by kayue** [comment]
Logging exception to log isn't a common practice in this bundle, but thanks anyway.

**Comment by kayue** [comment]
@peyman-mohamadpour Actually could you remove the logging part?
Just return `null` if meta is not found.

Because no meta found doesn't has to be an exception. It is one of the expected result. In fact if you use Doctrine's `getOneOrNullResult()` instead of `findOneBy()` method, you won't even get this exception at all.

**Comment by kayue** [comment]
@peyman-mohamadpour And if logging is indeed needed in your project, you can still do it by checking null.

```
if (wp_get_attachment_alt_text() == null) { $loggin->log('Error!'); }
```

**Pull request merged**